commit 71c8090207150a976c58ab6bdc73a70b8ecce9cd (refs/changes/22/4622/1)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2018-08-02T18:04:52+02:00 (1 year, 2 months ago)
    
    TOOLS-1964 sdc's zone SAPI template should use RabbitMQ domain instead of hardcoded IP

diff --git a/sapi_manifests/sdc/template b/sapi_manifests/sdc/template
index 9f4684f..c00213b 100644
--- a/sapi_manifests/sdc/template
+++ b/sapi_manifests/sdc/template
@@ -11,7 +11,7 @@
     "mahi_domain": "mahi.{{{datacenter_name}}}.{{{dns_domain}}}",
     "napi_domain": "{{{napi_domain}}}",
     "papi_domain": "{{{papi_domain}}}",
-    "rabbitmq": "{{{rabbitmq}}}",
+    "rabbitmq": "guest:guest:{{{rabbitmq_domain}}}:5672",
     "sapi_domain": "{{{sapi_domain}}}",
     "sdc_key_id": "{{{SDC_KEY_ID}}}",
     "ufds_admin_login": "{{{ufds_admin_login}}}",
